doc/Analytics.xml

```diff
@@ -4453,6 +4453,131 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Object Track Aggregation</title>
+      <para>This event is intended to describe an object throughout visible duration within a scene.
+        The description should include aggregated or summarized information along with contextual
+        data, such as images or bounding boxes, to assist clients in visualizing the object's
+        trajectory. This event can complement per-frame, per-object scene description by providing a
+        higher-level representation of the object’s trajectory, enhancing forensic analysis while
+        optimizing bandwidth consumption.</para>
+      <para>It is recommended to enable ObjectTrack data in sparsely crowded scenes to optimize data
+        produced from the device.</para>
+      <para>The process by which device aggregates object track data for a given object is out of
+        scope of this specification.</para>      
+      <para>The Object Track Aggregation rule generates an object track aggregation event for below
+        listed scenarios </para>
+      <itemizedlist>
+        <listitem>
+          <para> Optionally, Initial aggregation after an object appears OR an Intermediate
+            aggregation while the object is present in the field of view.</para>
+        </listitem>
+        <listitem>
+          <para> Final aggregation when the object is no longer visible in the field of view. </para>
+        </listitem>
+      </itemizedlist>
+      <para>Optionally, device may include additional object track data for e.g. snapshot image as
+        configured in ObjectTrackDataFilter parameter.</para>
+      <para>If an initial or intermediate aggregation is sent for an object, Final aggregation event
+        for that object shall be sent even if final classification does not meet class or threshold
+        filters in rule configuration.</para>
+      <para>Device should fill in aggregated tt:Object structure with data qualified as
+        classifications for e.g. Vehicle or License plate and should not fill in temporal data like
+        speed etc., as they are not meaningful in the aggregation context.</para>
+      <para>In order to reinforce aggregation data, Device should use ObjectTrack tt:Object
+        structure to provide interesting observations captured at various time instances within the
+        track duration like cropped object image etc.,</para>
```

Comment:
Can be nice if its mentioned, that data in objecttrack can be sparse.

Reply:
I can add a text to reflect your comment.

Reply:
updated text as per feedback.